### DISCUSSION TITLE
Adding KEF LSX LT II

### DISCUSSION THREAD

**Discussion by SLTWT** [pierreaubert/spinorama#298] [General]
KEF LSX LT II
https://www.erinsaudiocorner.com/loudspeakers/kef_lsx_ii_lt/

Thank you!

**Comment by pierreaubert** [comment]
https://www.spinorama.org/speakers/KEF%20LSX%20II%20LT/ErinsAudioCorner/index_eac.html

You are in luck, Erin sent me the data this weekend.
